packages/components/Molecules/Layout/BaseLayout.vue

```diff
@@ -1,6 +1,5 @@
 <template>
   <div class="rpl-site-layout">
-    <section><slot name="alert"></slot></section>
```

Comment:
I believe removing this will mean alerts wont render! Maybe the section needs to be moved to `rpl-alert` or render the section tag only if there is `$slots.alert` content. The section tag is important so the alert is an ARIA landmark.

Reply:
The alerts is currently placed in the header slot, and I couldn't find anything using this slot. As discussed I will add the slot back in, but remove the surrounding section tag to fix the validation issue. Alerts can be handled separately.